src/cpp/client.cpp

```diff
@@ -223,6 +223,8 @@ void Client::put_tensor(const std::string& key,
             tensor = new Tensor<uint8_t>(p_key, data, dims,
                                          type, mem_layout);
             break;
+        default :
+	    return; // FINDME: Need to handle this case better
```

Comment:
For these default cases in the C++ client, should we throw ``std::runtime_error`` to be consistent with all of the other invalid inputs or cases throughout the code?

Reply:
I can do that as a temporary measure, though we're going to have to switch away from exceptions as our error handling scheme longer term if we want to properly support C and Fortran clients.  We need to have a group discussion on this topic after the release.